src/v2/Components/Onboarding/Steps/Budget.tsx

```diff
@@ -1,18 +1,38 @@
-import React from "react"
+import React, { useState } from "react"
+import { ContextModule, OwnerType } from "@artsy/cohesion"
 import { commitMutation, graphql } from "react-relay"
 import styled from "styled-components"
 import { ProgressIndicator } from "v2/Components/ProgressIndicator"
 import { BudgetUpdateMyUserProfileMutation } from "v2/__generated__/BudgetUpdateMyUserProfileMutation.graphql"
-import { SystemContextProps, withSystemContext } from "v2/Artsy"
+import { withSystemContext } from "v2/Artsy"
 import Colors from "../../../Assets/Colors"
 import { MultiButtonState } from "../../Buttons/MultiStateButton"
 import { media } from "../../Helpers"
-import { Props } from "../Wizard"
 import SelectableToggle from "../SelectableToggle"
-import { StepProps } from "../Types"
 import { Layout } from "./Layout"
-import track from "react-tracking"
-import Events from "../../../Utils/Events"
+import { useTracking } from "v2/Artsy/Analytics/useTracking"
+
+const updateUserProfile = (priceRangeMax, relayEnvironment) => {
```

Comment:
(non blocking) would be nice to add some types to args

Reply:
Thanks for the nudge! With 5396c25 I extracted `Props` interfaces for these two steps and that caused me to revisit these updater funcs and learn how to type `relayEnvironment`. 😍